### PR TITLE
Use shields.io badges

### PR DESCRIPTION
**Pull request by epruesse**
Need to test this manually yet (see how it looks)

### CODE REVIEW

**Comment by jakirkham** [comment]
Toggling to refresh CI.

**Comment by jakirkham** [comment]
Have you had a chance to play with this yet, @epruesse? Need any pointers?

**Comment by epruesse** [comment]
@jakirkham Sadly no. Do you have a setup where you can test it easily?

**Comment by isuruf** [comment]
Works for me

**Comment by isuruf** [comment]
Thanks for the PR

File: conda_smithy/templates/README.md.tmpl [hunk 1]
```diff
@@ -16,40 +16,40 @@ Summary: {{ package.meta.about.summary }}
...
 {%- else %}
-Linux: ![]({{ disabled_badge_url }})
+![Linux disabled]({{ shield }}badge/linux-disabled-lightgrey.svg)
 {%- endif %}
 {%- if travis.enabled %}
-OSX: [![TravisCI](https://travis-ci.org/{{ github.user_or_org }}/{{ github.repo_name }}.svg?branch=master)](https://travis-ci.org/{{ github.user_or_org }}/{{ github.repo_name }})
+[![OSX]({{ shield }}travis/{{ github.user_or_org }}/{{ github.repo_name }}.svg?label=OSX)](https://travis-ci.org/{{ github.user_or_org }}/{{ github.repo_name }})
 {%- else %}
-OSX: ![]({{ disabled_badge_url }})
+![Linux disabled]({{ shield }}badge/OSX-disabled-lightgrey.svg)
```
Comment (jakirkham) [inline conda_smithy/templates/README.md.tmpl]: Noticed a small typo here. Fixing in PR ( https://github.com/conda-forge/conda-smithy/pull/728 ).

**Comment by jakirkham** [comment]
Just wanted to say thanks again for doing this. 😄

The responsiveness of these badges alone is a major improvement. Watched a badge bump the version within seconds of the package being published.

Yay for CDNs! 🎉

**Comment by epruesse** [comment]
:)

**Pull request merged**